debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -165,6 +165,9 @@ public void shouldConsumeAllEventsFromDatabase() throws SQLException, Interrupte
         assertInsert(updates.get(0), "id", 2001);
         assertDelete(updates.get(1), "id", 1001);
         assertTombstone(updates.get(2), "id", 1001);
+        
+        //Testing.Print.enable();
```

Comment:
I've intentionally left these as I often want to enable printing things when running the tests, and it's far easier to uncomment these than repeatedly figure out the best place to put them.